src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -20,9 +21,11 @@ public sealed partial class QuicConnection : System.IDisposable
         public IPEndPoint RemoteEndPoint => throw null;
         public QuicStream OpenUnidirectionalStream() => throw null;
         public QuicStream OpenBidirectionalStream() => throw null;
+        public long GetMaximumBidirectionalStreamCount() => throw null;
+        public long GetMaximumUnidirectionalStreamCount() => throw null;
```

Comment:
We'll want to tweak the names for these; each side has their own maximums, so something like remote vs local.

Also, we need an event on the connection to get MAX_STREAMS updates.

Reply:
Also I think we confirmed msquic these numbers are "available streams" not "max streams"?

Reply:
I had "Peer" in there originally, but it didn't feel great.

And yeah, I'll rename to available.